autocompleter/base.py

```diff
@@ -22,6 +22,8 @@
 EXACT_SET_BASE_NAME = AUTO_BASE_NAME + '.es'
 TERM_SET_BASE_NAME = AUTO_BASE_NAME + '.ts'
 RESULT_SET_BASE_NAME = 'djac.results.%s'
+FACET_SET_BASE_NAME = AUTO_BASE_NAME + '.f.%s'
```

Comment:
Yeah we need to store the facet key for each of these as well!!! so llif ".f.%s.%s" ... provider name, facet key, facet id